src/mbgl/map/tile_worker.cpp

```diff
@@ -144,10 +142,11 @@ void TileWorker::parseLayer(const StyleLayer* layer, const GeometryTile& geometr
                                      *collisionTile);
 
     std::unique_ptr<Bucket> bucket = layer->createBucket(parameters);
+    const SymbolLayer* symbolLayer = layer->as<SymbolLayer>();
```

Comment:
Maybe add a comment here that it's `null` when conversion fails, since it's not obvious that `.as<>` does a `dynamic_cast` internally.